src/main/java/com/book/store/app/service/BookService.java

```diff
@@ -7,4 +7,6 @@ public interface BookService {
     Book save(Book book);
 
     List<Book> findAll();
+
+    Book findById(Long id);
```

Comment:
Your service should work now with DTO's instead:
```suggestion
BookDto save(CreateBookRequestDto dto);

List<BookDto> findAll();

BookDto findById(Long id);
```